content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,269 @@
+performant releases -- that _also_ perform the same function as commercial
+instrumentation agents.
+
+### What's Changing?
+
+There's three major changes that the project will be rolling out. Keep in mind
+the exact details of some of this may change, but these goals are more or less
+fixed.
+
+1. All components across all repositories (including semantic conventions)
+   should adhere to a consistent way of communicating stability, through a
+   metadata file/information, that can be discovered and parsed in a
+   programmatic way. The exact format will be defined through an OTEP and
+   incorporated into the specification.
+2. Stability requirements will be expanded to include more requirements around
+   documentation and where it's hosted, example code, performance benchmarks
+   (where applicable), implementation cookbooks, and other artifacts as
+   necessary.
+3. Stable distributions of OpenTelemetry should only enable stable components by
+   default. Users should be able to select a desired minimum stability level
+   with a documented and consistent configuration option.
+
+This is a big change for maintainers, especially those who have shipped v1+ of
+their libraries. We would deeply appreciate your feedback on this proposal in
+the [discussion](https://github.com/open-telemetry/community/discussions/3098).
+
+## 2. Instrumentation Stability and Semantic Conventions
+
+As mentioned above, we would like to adjust the instrumentation stability
+process in order to provide greater consistency across projects, but also to
+better communicate the interplay between telemetry stability and implementation
+stability.
+
+### Background
+
+Semantic conventions evolve slowly and deliberately because they must work
+across diverse telemetry systems. While OpenTelemetry is designed for
+interconnected signals flowing together, users deploy many different storage and
+analysis engines to consume this data. Each backend has its own constraints and
+capabilities. Maintainers must balance competing concerns – keeping cardinality
+manageable, ensuring attributes are useful but not overly specific, and making
+conventions that work well regardless of where the data ends up.
+
+The downside of this is that progress on semantic conventions can be slow, and
+this slowness impacts all consumers of the conventions. Many instrumentation
+libraries are currently stuck on pre-release versions because they depend on
+experimental semantic conventions. Outside contributors are stuck between
+emitting unspecified telemetry or trying to engage in the process, which
+requires a long commitment. Finally, we're internally inconsistent in
+instrumentation across the project; Some libraries are mapped to conventions,
+others exist independently of it.
+
+### What's Changing?
+
+Our goals here are designed to achieve three outcomes.
+
+1. Instrumentation stability should be decoupled from semantic convention
```

Comment:
I am not sure if this is a good point for end users. The end users depend on semantic conventions in their visualization and alerting tools. 

As a user updating the instrumentation library, I am very interested in knowing if the instrumentation changed any attributes, because it can break my dashboarding and alerting.

Reply:
Shall it be mentioned in this document?